### ISSUE TITLE
Errors/warnings in cosole

### ISSUE THREAD

**Issue by ergonomicus** [evercode1/view-maker#2]
When changed to non production version of vue.js (<script src="https://cdnjs.cloudflare.com/ajax/libs/vue/1.0.21/vue.js"></script>), meaning not minified, I get bunch of vue.js errors in console, like 'Invalid prop: type check failed for prop "data"...' or 'Error when evaluating expression...'.

This is happening on widget index page, widget-grid component...

**Comment by evercode1** [comment]
I was able to remove 4 invalid prop typos by changing the type cast on the component to null.  I pushed that change and you can get rid of those warnings by installing the latest version of ViewMaker.

I didn't have any luck with the 9 other warnings, which seem really odd to me. 

For example:

Error when evaluating expression "row.Id": TypeError: Cannot read property 'Id' of null (found in component: <widget-grid>)

If it can't evaluate it, then how am I getting the correct output, in spite of the warning?  If row is null, then how are we getting a value for it?

I will look further into this, so I will leave this issue open.  If anyone has any ideas on the problem, I would love to hear it.  I'm new with using vue.js.

**Comment by evercode1** [comment]
Ok, update to ViewMaker 3.2.5 and it's all good, the warnings are gone.  Had to change a couple of property types and fix a typo.

**Comment by ergonomicus** [comment]
Sorry, I'm giving you a hard time, but I think you should initialize first_page_url and last_page_url to an empty string '' (in 'bootstrap the vue instance' part). Also go_to_page should be integer (I would put 1 there).

Besides that, I'm getting this also

Error when evaluating expression "sortOrders[key] > 0 ? 'asc' : 'dsc'": TypeError: Cannot read property 'Id' of undefined (found in component: widget-grid)

and couple more...

**Comment by evercode1** [comment]
It's not a hard time, I appreciate you looking at this.  The errors you are getting indicates that you do not have version 3.2.5 installed.  Those problems are solved as of that version.

Even though all the errors were solved, your suggestion about initializing the properties is probably right and  will make it better,.  I've already tested it and will get it out today for version 3.2.6.  Thanks again.
